business_requirement/views/business_view.xml

```diff
@@ -61,6 +61,12 @@
                                 <group col="4">
                                     <field name="parent_id"/>
                                     <field name="level"/>
+                                    <field name="creation_date"/>
```

Comment:
You might want to have 2 columns. eg:

Created by : Eric   |on: 2016/02/20
-------------------------|------------------------
Confirmed by : Eric   |on: 2016/02/20
-------------------------|------------------------
Approved by : Eric   |on: 2016/02/20
-------------------------|------------------------

(table is not that clean but I hope you understand the idea.)